radar-upload-frontend/src/components/Patient/PatientFilter.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <div>
+    <v-text-field
+      v-model="searchText"
+      class="mt-4"
+      prepend-inner-icon="mdi-magnify"
+      label="Search the patients"
```

Comment:
Maybe use the term `participants` in the texts that will be visible to the user.

Reply:
Do we have official docs or some things for these labels? Since there a lot of other texts I just assumed that may need to change all at once.

Reply:
Yes. That is true. This is the easiest I can find. https://radar-base.org/index.php/2019/02/13/concepts-in-management-portal/ though it is very generic. We can maybe use the existing term `subjects` or otherwise `participants`. @blootsvoets WDYT?

Reply:
yes participants seems apt.